static/templates/left_sidebar.hbs

```diff
@@ -93,4 +93,11 @@
             </div>
         </div>
     </div>
+    <div class="only-visible-for-spectators">
+        <a id="login-link">
+            <i class="fa fa-plus-circle" aria-hidden="true"></i>
+            {{~!-- squash whitespace --~}}
+            {{t "Log in to browse more streams" }}
+        </a>
+    </div>
```

Comment:
I don't think this is the correct way to place this -- I think it belongs in the same position in the DOM as the `subscribe-to-more-streams` block.